docs/store.md

```diff
@@ -169,6 +174,7 @@ The store specification defines 3 operations ```LISTREFERRERS```, ```GETBLOB```,
 
 **```GETREFMANIFEST```**: The content of the reference manifest is returned as byte array via ```stdout``
 
+**```RESOLVETAG```**: The digest that is resolved is returned as a byte array via ```stdout```
```

Comment:
Could we return the descriptor. Basically the mediaType and size might be needed for all other purposes. Digest alone might need extending and instead depending on the descriptor makes its an object which is expandable through properties later on if needed.

Reply:
This resolution happens only if subject is referenced by tag. If it is referenced by digest, this resolution is skipped. So we might not have the descriptor in all cases. I think we can change this to GetSubjectDescriptor which will return the descriptor irrespective of subject referenced by digest or tag

Reply:
This can be used by notaryv2 when verifying the signed payload

Reply:
I replaced resolve tag with get subject descriptor as it might be useful when doing notaryv2 verification. I will log another issue for using this descriptor information in notaryv2